.rubocop.yml

```diff
@@ -13,6 +13,11 @@ inherit_mode:
   merge:
     - Exclude
 
+Style/NumericLiterals:
+  Exclude:
+    - spec/migrations/convert_address_to_uk_address_spec.rb
+    - spec/migrations/convert_date_to_other_date_spec.rb
+    - spec/migrations/convert_old_text_fields_to_new_text_fields_spec.rb
```

Comment:
Might be worth excluding anything in the `spec/migrations` folder something like `spec/migrations/**/*' should do it rather htan manually ignoring every file that will use a version id

Reply:
Done, makes sense to me!